test/lib/usd/translators/UsdImportPreviewSurface/world.usda

```diff
@@ -0,0 +1,7 @@
+#usda 1.0
+
+over "refSphere" (
+    prepend references = @props/billboards/billboards.usda@
+)
```

Comment:
The test requires referenced assets since we want to make sure the rebasing works. I added a second layer with a few overs to add more composition arcs.